packages/Webkul/Shop/src/Resources/lang/en/app.php

```diff
@@ -845,6 +845,17 @@
                 'password'                => 'Password',
             ],
 
+            'admin' => [
+                'registration' => [
+                    'dear'         => 'Dear Admin',
+                    'description'  => 'A new customer has registered in the system.',
+                    'name'         => 'Customer Name: :customerName',
+                    'subject'      => 'New Customer Registration',
+                    'thanks'       => 'Thank you.',
+                ],
+
+            ],
+
```

Comment:
Please translate the all other langs you can take help of google translation or any other tool.